.github/workflows/main.yml

```diff
@@ -95,12 +95,6 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-buildx-
 
-      - name: Login to Docker Hub
```

Comment:
Docker login is good to keep to avoid the pull limits. If needed, I can add my user account here, or if you can access the repository settings you can add your own.

Reply:
See https://github.com/docker/login-action#docker-hub

Reply:
Sorry yes I temporarily removed it because I don't have access and was going to add it back when I get access. Yes please if you would be able to add your account that would be great.

Reply:
Alright, I've given you access and added the secrets.

Reply:
Thanks.